onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4208,19 +4208,50 @@ def aten_native_dropout_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::native_group_norm", trace_only=True)
 def aten_native_group_norm(
-    input: TensorType,
-    weight: Optional[TensorType],
-    bias: Optional[TensorType],
-    N: INT64,
-    C: INT64,
-    HxW: INT64,
-    group: int,
-    eps: float,
-) -> tuple[TensorType, TensorType, TensorType]:
+    input: TFloat,
+    weight: Optional[TFloat],
+    bias: Optional[TFloat],
+    N: INT64 = None,  # pylint: disable=unused-argument
+    C: INT64 = None,  # pylint: disable=unused-argument
+    HxW: INT64 = None,  # pylint: disable=unused-argument
+    group: int = None,
+    eps: float = None,
+) -> TFloat:
+    # FIXME: for the return, we can only return one TReal instead of [x,y,z]
+    # Because we don't how to computer the running_var and running_mean
+    # No native_group_norm test case, and the group_norm function in torch only return one output
```

Comment:
We do need to return three things because that is what the aten op does. I would skip the tests if we need to